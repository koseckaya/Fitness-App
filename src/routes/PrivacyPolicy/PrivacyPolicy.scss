@import "src/styles/vars";

.scl-privacy-policy {
  box-sizing: border-box;
}
