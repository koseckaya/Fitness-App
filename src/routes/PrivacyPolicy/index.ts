export { default as PrivacyPolicy } from './PrivacyPolicy';
export type { Props as PrivacyPolicyProps } from './PrivacyPolicy';
