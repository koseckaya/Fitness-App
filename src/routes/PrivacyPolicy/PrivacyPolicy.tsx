import { FC} from 'react';
import './PrivacyPolicy.scss';

export type Props = {
    className?: string;

};

const PrivacyPolicy: FC<Props> = ({ className }: Props) => (
    <div
        className={`privacy-policy ${className}`}
    >
        <span>PrivacyPolicy</span>
    </div>
);

export default PrivacyPolicy;
