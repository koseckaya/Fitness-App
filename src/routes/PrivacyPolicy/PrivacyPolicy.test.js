import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';
import PrivacyPolicy from './PrivacyPolicy';


it('renders correctly', () => {
    const tree = renderer
        .create(<Router><PrivacyPolicy /></Router>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});