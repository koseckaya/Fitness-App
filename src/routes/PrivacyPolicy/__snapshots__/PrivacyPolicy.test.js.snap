// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="privacy-policy undefined"
>
  <span>
    PrivacyPolicy
  </span>
</div>
`;
